.github/workflows/ci.yml

```diff
@@ -38,10 +44,44 @@ jobs:
           python -m pip install -r documentation/library_guide/requirements.txt
       - name: Build documentation
         run: |
-          mkdir html 
+          mkdir html
           sphinx-build -b html documentation/library_guide/ html/
       - name: Archive build directory
         uses: actions/upload-artifact@v1
         with:
             name: onedpl-html-docs-${{ env.GITHUB_SHA_SHORT }}
             path: html
+
+  linux-host-testing:
+    name: ${{ matrix.os }}_${{ matrix.cxx_compiler }}_cxx${{ matrix.std }}_${{ matrix.build_type }}
+    runs-on: ['${{ matrix.os }}']
+    timeout-minutes: 20
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-18.04
+            cxx_compiler: g++
+            std: 17
+            build_type: release
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get oneTBB
+        run: |
+          wget ${ONETBB_LINUX_LINK}
+          tar xf oneapi*
+      - name: Run testing
+        shell: bash
+        run: |
+          set -x
+          source oneapi*/env/vars.sh intel64
+          if [[ "${{ matrix.build_type }}" == "debug" ]]; then
```

Comment:
Do you think it makes sense to add it now without debug configuration enabled?

Reply:
Yes, as we may forget to add this check in the future.

Reply:
But we may not need it in future at all, or you sure we will?

Reply:
I'm not sure. Ok I removed this